include/deal.II/differentiation/ad/ad_drivers.h

```diff
@@ -838,29 +1207,73 @@ namespace Differentiation
 
```

Comment:
I can't quite attach this comment to the place where I want it, namely these lines:
```
     template <typename ADNumberType>
     struct TapedDrivers<
       ADNumberType,
       double,
       typename std::enable_if<ADNumberTraits<ADNumberType>::type_code ==
                               NumberTypes::adolc_taped>::type>
     {
```

Why do you have to define this specialization? Can't you just fall back to the general template -- which should be the backup for all specializations?

Reply:
Oh, I missed that. Ignore it then!